Source/Constraint.swift

```diff
@@ -61,6 +61,11 @@ public class Constraint {
     internal var makerFile: String = "Unknown"
     internal var makerLine: UInt = 0
     
+    public var layoutConstraint: LayoutConstraint? {
```

Comment:
This isn't a valid API because a Constraint can represent more than 1 LayoutConstraint and by not exposing it we can ensure SnapKit always has total ownership over the LayoutConstraints.